magprime/site_sections/magprime_reports.py

```diff
@@ -54,7 +53,8 @@ def sweatpants_counts(self, session):
         sweatpants_attendees = session.query(Attendee).filter(Attendee.amount_extra >= c.SUPPORTER_LEVEL)
 
         for key, val in c.SWEATPANTS_OPTS:
-            counts['Size unknown' if key == c.NO_SWEATPANTS else val] = sweatpants_attendees.filter(Attendee.sweatpants == key).count()
+            counts['Size unknown' if key == c.NO_SWEATPANTS else val] = \
+                sweatpants_attendees.filter(Attendee.sweatpants == key).count()
```

Comment:
Line too long, so I split it up.  FWIW, I would be in favor of raising the max line length significantly, but that's just a personal preference.